samples/bluetooth/peripheral_bms/README.rst

```diff
@@ -58,8 +58,9 @@ After programming the sample to your development kit, test it by performing the
 
    a. Click the :guilabel:`Settings` button for the device in nRF Connect.
    b. Select :guilabel:`Pair`.
-   c. Check :guilabel:`Perform Bonding`.
-   d. Click :guilabel:`Pair`.
+   c. Select :guilabel:`Keyboard and display` in the IO capabilities setting.
+   d. Check :guilabel:`Perform Bonding`.
```

Comment:
Maybe select instead of "check". Or if there is a check box in the UI, then It might be correct. Haven't checked the UI myself.

Reply:
In the UI, there is an empty box for checking. This line is nearly the same as before the change

Reply:
Okay, our "bible" Read Me First!" book suggests you should use "select" with the checkboxes. Not a big deal if you don't want to change it, but just for the consistency. We seem to have a couple of instances in the current NCS docs with select.

Reply:
fixed